src/main/kotlin/pl/edu/prz/kod/adapters/http/Events.kt

```diff
@@ -0,0 +1,24 @@
+package pl.edu.prz.kod.adapters.http
+
+import org.http4k.core.Uri
+import org.http4k.events.AutoMarshallingEvents
+import org.http4k.events.Event
+import org.http4k.events.EventFilters
+import org.http4k.events.then
+import org.http4k.format.Jackson
+import pl.edu.prz.kod.domain.Language
+
+val logEvent =
+    EventFilters.AddTimestamp()
+        .then(EventFilters.AddEventName())
+        .then(EventFilters.AddZipkinTraces())
+        .then(AutoMarshallingEvents(Jackson))
+
+data class IncomingHttpRequestEvent(val uri: Uri, val status: Int, val duration: Long) : Event
```

Comment:
I feel like the `Incoming` word is a bit misleading here, as at first I thought it would represent the event where request just came in (and in such case status and duration didn't make sense). I would suggest to remove it and just make it `HttpRequestEvent`.